src/annotator/components/NotebookModal.tsx

```diff
@@ -120,11 +120,7 @@ export default function NotebookModal({
     >
       <div className="relative w-full h-full" data-testid="notebook-inner">
         <div className="absolute right-0 m-3">
-          <IconButton
-            title="Close the Notebook"
-            onClick={onClose}
-            variant="dark"
-          >
+          <IconButton title="Close the Notebook" onClick={onClose}>
```

Comment:
I think we can find a styling middle ground here! What we can do is:

* Make it so the button does not have a background color when "at rest", so that it does not interfere with modal overlays
* But still use the `dark` variant color scheme overall, including the visually-helpful hover background color.

We could do this:

```suggestion
          <IconButton
            title="Close the Notebook"
            variant="dark"
            onClick={onClose}
            classes={classnames(
              // Remove the dark variant's background color to avoid
              // interfering with modal overlays. Re-activate the dark variant's
              // background color on hover.
              // See https://github.com/hypothesis/client/issues/3676
              '!bg-transparent enabled:hover:!bg-grey-3'
            )}
          >
```

I wish we could use a single `!important` rule but AFAIK there are no Tailwind utility classes for "not-hovered", so here we turn off all backgrounds and then turn them back on for one specific case.

Of course we want to use `!important` rules very sparingly, but I think this is the proper use case: this is indeed a hack and we want the overriding `!important` rules to stand out. I also think it's a good idea to leave breadcrumbs behind about _why_ we're manually overriding here, ergo the link to the original issue in comments.

The different variants and how `classes` works with components is document in the `frontend-shared` package's pattern library (including a page outlining how to customize or override component styling) — let me know if I can guide further or if any of that it completely non-sensical!

Reply:
Thanks!

Reply:
I like the approach. Pushed!